src/db/tables/subscriber_churns.ts

```diff
@@ -32,4 +32,18 @@ async function upsertSubscriberChurns(
   }
 }
 
-export { upsertSubscriberChurns };
+async function getAllSubscriberChurns(): Promise<SubscriberChurnRow[]> {
+  try {
+    const res = await knex("subscriber_churns").select().returning("*");
```

Comment:
AFAIK [`returning` is for inserts, updates and deletes](https://knexjs.org/guide/query-builder.html#returning):

```suggestion
    const res = await knex("subscriber_churns").select("*");
```

I think this might work as well somehow and we're already doing it elsewhere, but I just submitted [a PR](https://github.com/mozilla/blurts-server/pull/5521) that removes some instances that have both `select` arguments _and_ `returning`, so it's probably best to only use one?